### PR TITLE
[REVIEW] init change to fix #504

### LINKED ISSUE #504: cudf merge printing out dtypes
## Reporting a bug
cudf *merge* API printing out dtypes for all columns in jupyter notebook
System: Haswell 4xV100
Container: nvcr.io/nvidia/rapidsai/rapidsai:cuda9.2_ubuntu16.04

This happens on both of these cudf merges.  
```
    acq_df = acq_df.merge(names_df, how='left', on=['seller_name'])
```
Unexpected output below the notebook code cell.
```
1:int64
1:int32
1:float64
1:int64
1:int64
1:datetime64[ms]
1:datetime64[ms]
1:float64
1:float64
1:float64
1:float64
1:float64
1:int32
1:int32
1:int32
1:int64
1:int32
1:int32
1:int64
1:float64
1:int32
1:float64
1:float64
1:int32
2:int32
3:int32
```

- [ ] I am using the latest version of cuDF from conda, built from `master`, or
built from the latest tagged release. 
- [ ] I have included the version or commit hash I am using for reference.
- [ ] I have included the following environment details:
      Linux Distro, Linux Kernel, GPU Model
- [ ] I have included the following version information for:
      Arrow, cmake, CUDA, gcc/g++, Numpy, Pandas, Python
- [ ] I have included below a minimal working reproducer (if you are unsure how
 to write one see http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports).

### PR DESCRIPTION
**Pull request by AK-ayush**
This PR fix the unnecessary column dtypes printing(#504 ) while invoking the `DataFrame.merge()`.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: python/cudf/tests/test_joining.py [hunk 1]
```diff
@@ -267,3 +267,32 @@ def test_dataframe_multi_column_join():
         .sort_values(list(pddf_joined.columns))
         .reset_index(drop=True),
         pddf_joined)
+
+
+def test_dataframe_merge_stdout():
```
Comment (kkraus14) [inline python/cudf/tests/test_joining.py]: We can remove this test, this slipped through the cracks and will eventually be caught by more rigorous code linting.

**Comment by kkraus14** [comment]
run tests

File: python/cudf/tests/test_joining.py (hunk 1 above)
Comment (AK-ayush) [reply python/cudf/tests/test_joining.py]: removed the `test_dataframe_merge_stdout()` as suggested.

**Comment by kkraus14** [comment]
run tests

**Comment by kkraus14** [comment]
@AK-ayush need a changelog update and this will be good for merge, see the CHANGELOG.md file.

**Comment by kkraus14** [comment]
run tests

**Comment by kkraus14** [comment]
run tests

**Comment by kkraus14** [comment]
run tests

**Pull request merged**